### ISSUE TITLE
Tuning omega in range seperated functionals for TDDFT

### ISSUE THREAD

**Issue by jjren** [pyscf/pyscf#640]
I found that tuning omega in the range separated functionals in TDDFT will give different results compared to Gaussian16 (and Qchem). I tried two functionals wB97XD and  cam-b3lyp. 

I use pyscf 1.7.3. Following is my input
```
from pyscf import gto, tddft, dft
from pyscf.dft import xcfun

verbose = 5
basis = "6-31g"

molA = gto.M(atom = """
 H 0 0 0.6
 H 0 0 0
        """,
        basis = basis, verbose=verbose)
mfA = dft.RKS(molA)
#mfA.xc = "HYB_GGA_XC_WB97X_D"
mfA.xc = "camb3lyp"
mfA._numint.libxc = xcfun
mfA.omega = 0.2
mfA.kernel()

tdA = tddft.TDDFT(mfA)
tdA.kernel()
```
The gs and excitation energy are
```
PySCF version 1.7.3

w97XD
        Gaussian            pyscf
w0.2(default)
DFT     -1.15305260284  -1.15305207893701
TDDFT   16.10951541          16.109524
        27.71114516          27.711191
        49.10310738          49.099341

                           
w0.1                       
DFT     -1.15715625395  -1.15715571774563
TDDFT    16.1700343     16.52109143 
          27.6276020     28.11310012
          49.0403100     49.24150174
          
          
CAM-B3LYP 
default  
        -1.14700273560  -1.14700272733279
         16.17657222     16.17657382
         28.06468636     28.06468978
         49.12112989     49.12113246
         
w0.2     
       -1.14327216868a   -1.1432721599135
       16.14837289        16.24807925 
       28.01968627        28.08812373 
       49.00854076        49.08687747 

```
The SCF results are the same, but the TDDFT results seem incorrect.

**Comment by sunqm** [comment]
Bug fixed in b3bb4e3

**Comment by jjren** [comment]
Thanks, Qiming.  The results are correct now.
